Library/Formula/speech-tools.rb

```diff
@@ -0,0 +1,23 @@
+class SpeechTools < Formula
```

Comment:
The Edinburgh prefix is historic. This release comes from CMU, and the package is also called just **speech-tools** in Debian.